frontend/src/components/SectionSimpleBar/SectionSimpleBar.js

```diff
@@ -1,26 +1,43 @@
+import React, { useRef } from 'react'
 import Box from '@mui/material/Box'
-import PropTypes from 'prop-types'
 import Typography from '@mui/material/Typography'
+import Skeleton from '@mui/material/Skeleton'
+import PropTypes from 'prop-types'
 
 import ExpandableContainer from 'components/ExpandableContainer'
 import SimpleBar from 'components/SimpleBar'
 import { LayoutMetricType } from 'utils/propTypes'
+import { useClientSize } from 'hooks/victory'
 
 const SectionSimpleBar = ({ layout, data }) => {
   const xAxisKey = layout.x?.key
   const yAxisKey = layout.y?.key
+  const ref = useRef()
+  const defaultContainerWidth = 500
+  const defaultContainerHeight = 250
+  const size = useClientSize(ref, defaultContainerWidth, defaultContainerHeight, 0.5)
+
+  if (!layout) {
+    return <Skeleton variant="rectangular" sx={{ borderRadius: 1 }} />
+  }
```

Comment:
Individual metrics shouldn't have to worry about showing skeletons for `!layout`. If the layout is unknown, there's a full page spinner blocking any metrics from rendering.